File: 1-easy/problems/1700-number-of-students-unable-to-eat-lunch/solution.ts
function countStudents(students: any, sandwiches: any): boolean | number | string | any {
  """
  :type students: List[int]
  :type sandwiches: List[int]
  :rtype: int
  """
  count = collections.Counter(students)
  for (i, s in enumerate(sandwiches)) {
      if (not count[s]) {
          break
      count[s] -= 1
  } else {
      i = (sandwiches).length
  return (sandwiches).length-i;
}

export { countStudents };

File: 1-easy/problems/1700-number-of-students-unable-to-eat-lunch/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def countStudents(self, students, sandwiches):
        """
        :type students: List[int]
        :type sandwiches: List[int]
        :rtype: int
        """
        count = collections.Counter(students)
        for i, s in enumerate(sandwiches):
            if not count[s]:
                break
            count[s] -= 1
        else:
            i = len(sandwiches)
        return len(sandwiches)-i
